build-tools/xaprepare/xaprepare/ConfigAndData/Configurables.MacOS.cs

```diff
@@ -5,6 +5,8 @@ namespace Xamarin.Android.Prepare
 	partial class Configurables
 	{
 		const string JetBrainsOpenJDKOperatingSystem = "osx-x64";
+		const string MicrosoftOpenJDKOperatingSystem = "macOS/x64";
+		const string MicrosoftOpenJDKFileExtension   = "tar.gz";
```

Comment:
Since `MicrosoftOpenJDKFileExtension` is shared between mac and Linux, it can go into `Configurables.Unix.cs`